### PR TITLE
Train the Train material + some restructuration

### PR DESCRIPTION
**Pull request by bebatut**
This PR:
- Adds several tutorials (disabled with the PR to give time to review them after a pilot workshop) about pedagogy, tutorial design, training delivery, mostly based on the ELIXIR-GOBLET Train the Trainers program 
- Renames `instructors` topic to `teaching` (to be similar to `contributing`)
- Moves the philosophies page as a tutorial
- Modifies `teaching` and `contributing` subtopics

### CODE REVIEW

File: topics/teaching/tutorials/assessment/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+>
+> - Split into groups of 4, depending on the number of participants.
+> - Discuss how you could collect information from learners at the beginning of a course or lesson and how you can use it.
+> - Write it in a notebook and discuss with the rest of the group.
+{: .hands_on}
+
+
+## Formative vs summative assessment
+
+![A cartoon figure showing the main difference between formative and summative assessment, when making soup. When the chef tastes the soup while preparing, it's formative assessment. When the guests taste the soup after its done, it's summative asssessment.](images/FormativeSummativeAssessment.png "Formative and summative assessment")
```
Comment (hexylena) [inline topics/teaching/tutorials/assessment/tutorial.md:86]: ugh amazing, good alt-text, **THANK YOU** :tada: :yellow_heart: :tada: :yellow_heart: :tada: :yellow_heart:

**Comment by shiltemann** [comment]
nice! I guess we should add a `redirect_from` tag to all the pre-existing tutorials moved from the contributing topic to teaching?

File: topics/teaching/metadata.yaml [hunk 2]
```diff
@@ -12,9 +12,18 @@ summary: |
...
     The Galaxy community via the Galaxy Training Network supports instructors and
     workshop organizers using with a series of recommandations (tutorials below) and
-    regular discussion meetings.<br>
+    regular discussion meetings.
 
-    You can additionally <a href="philosophies/">read about our training philosophies.</a>
+subtopics:
+  - id: pedagogy
+    title: "Pedagogical aspects"
+    description: "These will guide you on the pedagogy to deliver impactfully trainings"
```
Comment (shiltemann) [inline topics/teaching/metadata.yaml:20]: I think the subtopic heading will still show up on the topic page, even though all the tutorials for this subtopic are disabled, maybe add `(work in progress)` to the end of the sentence so it doesn't look weird? (tho I think you could also just enable them now, and make updates after your pilot workshop, but that's up to you)

Comment (bebatut) [reply topics/teaching/metadata.yaml:20]: I was also thinking about enabling the 5 main tutorials directly

**Comment by bebatut** [comment]
> nice! I guess we should add a `redirect_from` tag to all the pre-existing tutorials moved from the contributing topic to teaching?
> 
> ```
> redirect_from:
>      - /topics/contributing/tutorials/tutorial-name/tutorial
> ```
> 
> (and the philosophies page)

I normally did it (hopefully) everywhere

**Comment by bebatut** [comment]
The PR should be ready now :tada:

File: topics/teaching/metadata.yaml (hunk 2 above)
Comment (shiltemann) [reply topics/teaching/metadata.yaml:20]: go for it!

File: topics/teaching/index.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+layout: topic
+topic_name: teaching
+---
```
Comment (shiltemann) [inline topics/teaching/index.md:6]: should this have the `redirect_from` as well?

File: topics/teaching/metadata.yaml (hunk 2 above)
Comment (bebatut) [reply topics/teaching/metadata.yaml:20]: I did it yesterday 🙂

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
Whoo! :tada: Very very nice @bebatut and @fpsom !!

**Comment by bebatut** [comment]
Thanks for merging it

**Pull request merged**